service.subtitles.legendasdivx/service.py

```diff
@@ -0,0 +1,584 @@
+# -*- coding: utf-8 -*-
```

Comment:
Usually this file, as it is the entrypoint script should be the smallest possible. The reason is that this is the only file that is not compiled to bytecode by the python interpreter. If possible try to refactor your addon to allow for something like:

```python
import sys
import mymaincodebase

mymaincodebase.run(sys.argv)
```

Reply:
ok

Reply:
still working on this ...

Reply:
Done.